tests/e2e/specs/shopper/cart-checkout/account.test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../../utils';
+import { SIMPLE_PHYSICAL_PRODUCT_NAME } from '.../../../../utils/constants';
+
+/**
+ * External dependencies
+ */
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+import {
+	merchant,
+	setCheckbox,
+	openDocumentSettingsSidebar,
+} from '@woocommerce/e2e-utils';
+import { switchUserToAdmin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+// Defining block details
+const block = {
+	name: 'Checkout',
+	slug: 'woocommerce/checkout',
+	class: '.wp-block-woocommerce-checkout',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'Skipping Cart & Checkout tests', () => {} );
+}
+
+describe( 'Shopper → Checkout → Account', () => {
+	beforeAll( async () => {
+		await merchant.openSettings( 'account' );
+		//Enable create an account at checkout option
+		await setCheckbox( '#woocommerce_enable_checkout_login_reminder' );
+		//Enable login at checkout option
+		await setCheckbox(
+			'#woocommerce_enable_signup_and_login_from_checkout'
+		);
+		//Enable guest checkout option
```

Comment:
🙌🏼  great comments describing what these actions are for. Thanks!

Minor nitpick: please could there be a space after the `//` and a period at the end of the comment?